lms/templates/courseware/course_about.html

```diff
@@ -167,9 +167,8 @@
             reg_element_id = "reg_then_add_to_cart"
           %>
           <a href="${reg_href}" class="add-to-cart" id="${reg_element_id}">
-            ${_("Add {course.display_number_with_default} to Cart ({currency_symbol}{cost})")\
-              .format(course=course, currency_symbol=settings.PAID_COURSE_REGISTRATION_CURRENCY[1],
-                      cost=registration_price)}
+            ${_("Add {course.display_number_with_default} to Cart ({price})")\
```

Comment:
Instead of putting the unwieldy "course.display_number_with_default" as a variable, can't you do something like, `_("{course_name}").format(course_name=course.display_number_with_default)`